README.Rmd

```diff
@@ -15,69 +15,128 @@ knitr::opts_chunk$set(
 
 # GitStats
 
-Please be reminded, that this work is still IN PROGRESS!
-
 <!-- badges: start -->
+[![R-CMD-check](https://github.com/r-world-devs/GitStats/workflows/R-CMD-check/badge.svg)](https://github.com/r-world-devs/GitStats/actions)
+[![lifecycle](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
 <!-- badges: end -->
 
-The goal of GitStats is to search through multiple GitHub and GitLab platforms for statistics.
+***WORK IN PROGRESS***
+
+The goal of GitStats is to search through multiple GitHub and GitLab platforms for different statistics either by owners/groups of repositories, team members or searched code phrases.
 
 For the time being GitStats supports connections to public GitHub, enterprise GitHub and GitLab.
 
 ## Installation
 
-You can install the development version of GitStats like so:
+Development version:
 
-``` r
+```r
 devtools::install_github("r-world-devs/GitStats")
 ```
 
-## How to start
+## Start - set connections
 
-Start with setting your git connections: choose GitHub Client or GitLab Client class.
-Choose your owners (GitHub) or group/groups (GitLab). 
-Provide API url and a token.
+Start with setting your git connection.
 
 
 ```r
-my_github <- GitHubClient$new(
-  owners = "r-world-devs",
-  rest_api_url = "api.github.com",
-  token = Sys.getenv("GITHUB_PAT")
-)
+library(GitStats)
 
-my_gitlab <- GitLabClient$new(
-  groups = c("my_group", "my_second_group"),
-  rest_api_url = "your_api_url.com",
-  token = Sys.getenv("GITLAB_PAT)
-)
+my_gitstats <- GitStats$new()
```

Comment:
I think you could build a vignette with example usage, you could put some examples in there  and in general readme only focus on basics

Reply:
I added an issue to build vignette #37